source/includes/platform-limitations.rst

```diff
@@ -0,0 +1,7 @@
+- Amazon Linux 1, Debian 9, and macOS 10.14 aren't supported.
+- Red Hat Enterprise Linux (RHEL) 7, Amazon Linux 2, SUSE Linux
+  Enterprise Server (SLES) 12, and Ubuntu 18.04 support is deprecated
+  and might be removed in a later ``mongosh`` release.
+- If you must use ``Node.js`` 16 with ``mongosh``, install ``Node.js``
+  and then install ``mongosh`` through ``npm``. This installation
+  configuration might be removed during the lifetime of ``mongosh`` 2.x.
```

Comment:
Do you think it’s unambiguous enough here that it’s not the ability to install mongosh through npm by itself that could go away, but the ability to run an npm-installed mongosh with Node.js 16?

Also, would recommend to get a review from @gaurabaryal for this text :slightly_smiling_face:

Reply:
Adjusted text.